repository/a.json

```diff
@@ -665,6 +665,17 @@
 			]
 		},
 		{
+			"name": "Amxx Pawn",
+			"details": "https://github.com/evandrocoan/SublimeAmxxPawn",
+			"labels": ["language syntax", "amx mod x", "amxx pawn"],
```

Comment:
It would probably make sense to remove the last two labels, especially the one that repeats the package name.